client/src/featureform/resources.py

```diff
@@ -876,13 +885,14 @@ class Source:
     tags: list
     properties: dict
     variant: str
+    is_directory: bool = False
```

Comment:
this is fine for now, but i think eventually we want abstractions so we don't need all these checks

Reply:
Changed this to just check the type instead